File: 191. Number of 1 Bits/bitwise.py
class Solution:
  def hammingweight (self, n : int) -> int:
    return bin(n).count("1")
  
class Solution:
  def hammingweight (self, n : int) -> int:
    count = 0
    while n:
      count += &1
      n >>= 1
    return count

File: 338. Counting Bits/bitwise.cpp
class Solution{
public:
  vector<int> countBits(int n){
    vector<int> result(n + 1, 0);
    for(int i = 0; i <= n; ++i)
      result[i] = result[i - (i & -i)] + 1;
    return result;
  }
};

class Solution{
public:
  vector<int> countBits(int n){
    vector<int> result;
    for(int i = 0; i <=n; ++i)
      result.push_pack(bitset<32>(i).count());
  }
  return result;
};
